### ISSUE TITLE
Can't utilise remote air temp

### ISSUE THREAD

**Issue by tomtokic** [andylockran/heatmiserV3#68]
Hi Andy,

Not sure if this is an issue for the heatmiser plugin in home assistant or for this code here, but I am unable to utilise my remote air sensor to display temp.  Instead I get 25 degrees celsius.  I have been able to see it in the DCB, but can't seem to get it into Home Assistant.

**Comment by andylockran** [comment]
Yeah, this is my problem for not making it configurable.  I'm looking to update this library to make it more rigorous, so if you can get me a DCB snapshot and put it under #71 I can hopefully try and help.

Andy
